### PR TITLE
Remove Kogito profile to productize required components

### PR DESCRIPTION
**Pull request by almope**
Signed-off-by: Alberto Morales Pérez <almorale@redhat.com>

**Thank you for submitting this pull request**

**JIRA**: 
(https://issues.redhat.com/browse/RHPAM-3393)

**referenced Pull Requests**:
* https://github.com/kiegroup/kie-wb-common/pull/3541

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/pom.xml [hunk 1]
```diff
@@ -653,174 +805,6 @@
         </plugins>
       </build>
     </profile>
-
-    <!-- Profile to build GWT code and package WAR for Community builds -->
-    <profile>
-      <id>kogito</id>
-      <activation>
-        <property>
-          <name>!productized</name>
```
Comment (Ginxo) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/pom.xml:662]: ant the same here, what about using `kogitoProductized` :thinking:

**Review by Ginxo — CHANGES_REQUESTED** [review]
check my comment please

**Review by Ginxo — APPROVED** [review]
LGTM after internal discussion

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/pom.xml (hunk 1 above)
Comment (almope) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/pom.xml:662]: @Ginxo answered in the other thread: https://github.com/kiegroup/kie-wb-common/pull/3541#discussion_r558269910

**Review by AntStephenson — APPROVED** [review]

**Pull request merged**